spec/Section 7 -- Response.md

```diff
@@ -61,21 +61,20 @@ contains should indicate why no data was able to be returned.
 
 If the `data` entry in the response is present (including if it is the value
 {null}), the `errors` entry in the response may contain any field errors that
-were raised during execution. If field errors were raised during execution, it
-should contain those errors.
+were raised during execution. 
```

Comment:
This is a concrete change in the interpretation of the spec, if you wish to have it included please raise it as a separate PR so that it can be discussed separately.